x/tss/types/types.go

```diff
@@ -193,7 +193,7 @@ func (m MultisigInfo) GetKeys() []ecdsa.PublicKey {
 
 // GetPubKeysByValidator returns pub keys a validator submitted
 func (m MultisigInfo) GetPubKeysByValidator(val sdk.ValAddress) []ecdsa.PublicKey {
-	var pubKeys []ecdsa.PublicKey
+	pubKeys := []ecdsa.PublicKey{}
```

Comment:
This would return nil before which was inconsistent with the return value in `GetMultisigPubKeysByValidator`.

Reply:
well I would have changed the other return to nil, but not I big point of contention. For most intents and purposes nil and empty slice behave identical

Reply:
Switched to `nil`